README.md

```diff
@@ -205,6 +211,19 @@ The Diffie-Hellman group should have at least 1024 bit and can be created with t
 ```
 # openssl dhparam -out /etc/exim4/dhparam.pem 2048
 ```
+In the transport section, you need to modify the user and group to the exim-user `system_aliases`-transport (around line 800):
+```
+system_aliases:
+  driver = redirect
+  allow_fail
+  allow_defer
+  data = ${lookup{$local_part}lsearch{/etc/aliases}}
+  user = <exim-user>
+  group = <exim-group>
+  file_transport = address_file
+  pipe_transport = address_pipe
```

Comment:
1. done
2. you won't need the last four lines for the redirection. But if you don't redirect, aren't these settings required to deliver mails to system users? If you haven't set a redirect yet, will it result in an error?